ui/common/css/build/theme-all.scss

```diff
@@ -0,0 +1,9 @@
+@import '../theme/default';
+
```

Comment:
these concrete sources in common/css/build compile to the actual color variable css sources at public/css/theme-all.css, theme-dark.css, theme-light.css, and theme-transp.css.